translator/src/main/java/com/google/devtools/j2objc/Options.java

```diff
@@ -531,8 +534,8 @@ else if (arg.equals("--final-methods-as-functions")
     }
 
     if (shouldMapHeaders() && buildClosure) {
-      ErrorUtil.error(
-          "--build-closure is not supported with -XcombineJars or --preserve-full-paths");
+      ErrorUtil
+          .error("--build-closure is not supported with -XcombineJars or --preserve-full-paths");
```

Comment:
Revert this diff. Having the line break after the "(" is more in line with internal Google style.